src/main/java/com/exactpro/th2/readcsv/impl/ProtoCsvContentParser.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.readcsv.impl;
+
+import com.exactpro.th2.common.grpc.RawMessage;
+import com.exactpro.th2.readcsv.cfg.CsvFileConfiguration;
+import com.google.protobuf.ByteString;
+import java.util.Map;
+
+public class ProtoCsvContentParser extends AbstractCsvContentParser<RawMessage.Builder> {
+    public ProtoCsvContentParser(Map<String, CsvFileConfiguration> configurationMap) {
+        super(configurationMap);
+    }
+
+    @Override
+    RawMessage.Builder createMessageBuilder(byte[] body) {
+        return RawMessage.newBuilder().setBody(ByteString.copyFrom(body));
```

Comment:
If we are sure that those bytes won't be mutated (and we are sure in this case if I am not mistaken of course) we can use `UnsafeByteOperations.unsafeWrap` to create a ByteString